IsraelHiking.Web/src/application/services/routing.provider.ts

```diff
@@ -52,6 +57,120 @@ export class RouterService {
         }
     }
 
+    private async getRouteOutsideIsrael(latlngStart: LatLngAlt, latlngEnd: LatLngAlt, routinType: RoutingType): Promise<LatLngAlt[]> {
+        const json = {
+            directions_options: {
+                directions_type: "none"
+            },
+            locations:[
+                {
+                    lon: latlngStart.lng,
+                    lat: latlngStart.lat,
+                    type: "break"
+                },
+                {
+                    lon: latlngEnd.lng,
+                    lat: latlngEnd.lat,
+                    type: "break"
+                }
+            ],
+        } as any;
+        switch (routinType) {
+            case "Hike":
+                json.costing = "pedestrian";
+                json.costing_options = {
+                    pedestrian: {
+                        "use_ferry": 1,
+                        "use_living_streets": 0.5,
+                        "use_tracks": 0,
```

Comment:
```suggestion
                        "use_tracks": 0.4,
```

Make tracks slightly less favorable that paths

Reply:
I think this is irrelevant when "shortest" is used